examples/AcousticWave2D_codegen.py

```diff
@@ -78,14 +78,21 @@ def damp_boundary(damp):
         self.dm = dm
 
     def Forward(self):
-        fw = ForwardOperator(self.m, self.src, self.damp, self.rec, self.u, time_order=self.t_order, spc_order=self.s_order, cache_blocking=True, profile=True, auto_tune=True)
+        fw = ForwardOperator(self.m, self.src, self.damp, self.rec, self.u, time_order=self.t_order, spc_order=self.s_order, cache_blocking=True, profile=True)
         fw.apply()
-        return (self.rec.data, self.u.data)
+        return self.rec.data, self.u.data
+
+    # Forward with auto tuning enabled
+    def Forward_auto_tune(self):
```

Comment:
Its my first PR here so appologies for the lack of description. Travis is not happy because I have set and pushed all the flags for auto tuning in case you want to pull and see how it works without any extra effort. Travis finds all the unrecognised pragmas and thus complains. Gonna fix it later.